cabal-testsuite/PackageTests/LiterateHaskell/Suffixed/test.cabal

```diff
@@ -0,0 +1,10 @@
+cabal-version: 2.2
+name: test
+version: 0
+
+executable foo
+  main-is: Main.lhs.tex
+  ghc-options: -x lhs
```

Comment:
As said, I'm not fan of this.

`cabal check` should not warn about `Main.lhs.tex` only if `ghc-options: -x lhs` is present. And I don't know how to make this check robust in presence of conditionals etc.

Making symbolic links feels to be more straight forward.

I'm sorry.